proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -6650,13 +6650,13 @@ lemma set_mcpriority_bound_sc_tcb_at_cur_thread[wp]:
 
 lemma reorder_ntfn_bound_sc_tcb_at_cur_thread[wp]:
   "\<lbrace>\<lambda>s. bound_sc_tcb_at P (cur_thread s) s\<rbrace>
-   reorder_ntfn a
+   reorder_ntfn a b
```

Comment:
Could you use the variables names from the definition of `reorder_ntfn` (and for the other uses of this function and `reorder_ep`)?